repos/system_upgrade/common/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -83,27 +92,95 @@ def get_installed_pkgs():
     return installed_pkgs
 
 
-def _get_repositories_mapping():
+def _get_enabled_repoids():
+    """
+    Collects repoids of all enabled repositories on the source system.
+
+    :param repositories_facts: Iterable of RepositoriesFacts containing repositories related info about source system.
+    :return: Set of all enabled repository IDs present on the source system.
+    :rtype: Set[str]
+    """
+    enabled_repoids = set()
+    for repos in api.consume(RepositoriesFacts):
+        for repo_file in repos.repositories:
+            for repo in repo_file.data:
+                if repo.enabled:
+                    enabled_repoids.add(repo.repoid)
+    return enabled_repoids
+
+
+def _get_repositories_mapping(target_pesids):
     """
     Get all repositories mapped from repomap file and map repositories id with respective names.
 
+    :param target_pesids: The set of expected needed target PES IDs
     :return: Dictionary with all repositories mapped.
     """
-    repositories_mapping = {}
 
-    repositories_map_msgs = api.consume(RepositoriesMap)
+    repositories_map_msgs = api.consume(RepositoriesMapping)
     repositories_map_msg = next(repositories_map_msgs, None)
     if list(repositories_map_msgs):
-        api.current_logger().warning('Unexpectedly received more than one RepositoriesMap message.')
+        api.current_logger().warning('Unexpectedly received more than one RepositoriesMapping message.')
     if not repositories_map_msg:
         raise StopActorExecutionError(
-            'Cannot parse RepositoriesMap data properly',
+            'Cannot parse RepositoriesMapping data properly',
```

Comment:
Well this message is just plain wrong, never noticed it before though. While the details seems actually be the actually accurate error. O.o
NVM now, but just saying.

Reply:
@vinzenz  basically it's not expected you will see it at all. It's here more like just in case you are changing e.g. the model format or something and forget to update this particular actor, so you will see immadiately the problem with nice msg. But it's not expected you will ever see this msg for any master/brew build :)